components/tidb_query_datatype/src/codec/table.rs

```diff
@@ -30,6 +30,59 @@ pub const TABLE_PREFIX_KEY_LEN: usize = TABLE_PREFIX_LEN + ID_LEN;
 // the maximum len of the old encoding of index value.
 pub const MAX_OLD_ENCODED_VALUE_LEN: usize = 9;
 
+#[derive(Debug)]
+pub enum Handle<'a> {
+    Int(i64),
+    Common(Vec<BytesSlice<'a>>),
```

Comment:
Maybe `General` is a better name?

Reply:
How about using SmallVec instead of Vec to reduce allocation?

Reply:
Common is used in TiDB, we better be consistent.